ext/pcre/php_pcre.c

```diff
@@ -138,6 +127,29 @@ static void pcre_handle_exec_error(int pcre_code) /* {{{ */
 }
 /* }}} */
 
+static const char *php_pcre_get_error_msg(php_pcre_error_code error_code) /* {{{ */
+{
+    switch (error_code) {
+        case PHP_PCRE_INTERNAL_ERROR:
+            return "Internal error";
+        case PHP_PCRE_BAD_UTF8_ERROR:
+            return "Malformed UTF-8 characters, possibly incorrectly encoded";
+        case PHP_PCRE_BAD_UTF8_OFFSET_ERROR:
+            return "Malformed UTF-8 offset, possibly incorrectly encoded";
```

Comment:
Any ideas on how to phrase this to make more sense to the user, @nikic?

Perhaps something along the line of:
> Offset does not point to a valid UTF-8 continuation byte

Or
 > The offset did not correspond to the beginning of a valid UTF-8 code point

(Stolen from [owncloud/core](https://github.com/owncloud/core/blob/b62780961b30d28d2f1f8dcb448c9e3b6e973528/lib/private/Files/Filesystem.php#L639)

Reply:
The second message sounds good to me.

Reply:
Okay, to me too. I'll change it